Watchman.DomainModel/Settings/Services/ConfigurationService.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Threading.Tasks;
+using Watchman.Cqrs;
+using Watchman.DomainModel.Settings.Commands;
+using Watchman.DomainModel.Settings.Queries;
+
+namespace Watchman.DomainModel.Settings.Services
+{
+    public class ConfigurationService : IConfigurationService
+    {
+        public Configuration Configuration
+        {
+            get
+            {
+                if (_lastRefreshed < DateTime.Now.AddMinutes(-10))
+                {
+                    this.ReloadConfiguration();
```

Comment:
nie podoba mi sie to, że w bazie aktualizuje sie w chwili zmieniania 
troche mało to bezpieczne

Reply:
i troche podchodzi pod złamanie SRP

Reply:
trochę zmieniłem, zobacz czy teraz ok

Reply:
dla samego pobierania powiedzmy że jeszcze może być 
nie podoba mi sie komunikacja z bazą pod spodem `get` w propertiesie 
ale powiedzmy że to akceptuje